app/presenters/curation_concerns/generic_work_show_presenter.rb

```diff
@@ -78,5 +78,11 @@ def show_thumb_member_presenters
         end
       end
     end
+
+    def riiif_file_id
+      return unless member_presenters.present?
+      representative_presenter = member_presenters.select{ |pres| pres.id == representative_id }.pop
+      representative_presenter.riiif_file_id
+    end
```

Comment:
I'm not totally sure if there might be a quicker way to get the representative presenter than loading _all_ the member presenters and filtering them... maybe not though. 

I'm not a fan of the contortions we've got to go through to get all this stuff, and to keep track of what we should be sending where when. And the performance implications of loading all this stuff. But this may just be what the stack requires.